variable/technology/technologies.yaml

```diff
@@ -1089,3 +1089,30 @@ Rate Automatic Frequency Restoration Reserve|Electricity|Wind|OffShore:
    description: Percentage of the Maximum Active Power that can be devoted to Automatic Frequency Restoration Reserve
                 for a Wind OffShore power plant
    unit: '%'
+   
+# description of Demand flexibility potentials
+Number of Units|Electricity|Residential|Demand Flexibility:
+   description: Number of units of kind "simple Hydro reservoir "
+   unit: 
+   
+Maximum Active power|Electricity|Residential|Demand Flexibility:
+   description: Potential for demand response in a given region/time frame, 
+                expressed as % decrease in residential electricity demand
+   unit: %
+   
+Maximum Storage|Electricity|Residential|Demand Flexibility:
```

Comment:
I'm confused between the variable name "Maximum storage" and the description "Potential for demand dispatch"...

Reply:
Following Gils 2014 (https://doi.org/10.1016/j.energy.2014.02.019) we can conceptualize demand flexibility as an electricity storage unit. Maximium storage is akin to a storage unit pulling power from the grid, i.e. increasing demand at that point in time or 'dispatching demand'. I am trying to use the already defined higher level variables' nomenclature here.

Does that work, or would another conceptualization be preferred?

Reply:
It makes sense when you interpret flexibility as temporal shifting. Still, it would be good to clarify the description (clearly stating that this flexibility is interpreted as storage), and/or add a `note` or `seealso` attribute with the scientific reference (this is also the attribute used as [sphinx markup directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-seealso))

Reply:
I think the name should include 'shifting' instead of storage.. Modelling those as storages is a modelling concept but ther could be others. Regarding the demand response, we usually define 2 kinds: 
1/ load curtailement => this consists in "erasing" a certain part of the demand, which may be replaced by another source of local energy (eg local fuel generators which are used to compensate load curtailement which is used mainly for dealing with transmission problems), or not replaced at all (eg for individuals, they agree not to consume if asked to but in exchange get a more interesting price)
2/ Load shifting => this consists in 'moving' a certain amount of energy from one timeframe to another. Typical example for individuals is tariffs that incent them to run the washing machine at night and not during the peaks
I think this kind of vocabulary should be used

Reply:
Then we may describe the variables in the same spirit than for power plants : we have different kinds of power plants (eg coal, gas...) with the same set of characteristics (maximum power, reserve capabilities, costs....); Here we have different kinds of demand-response which could be load shifting / load curtailment; could apply to different segments of customers (segments are already defined in "energy" variables / and have specific characteristics : costs, amount of power that is curtailed/shifted, amount of energy that can be curtailed/shifted (in that case it is close to a storage)...